libs/core/src/lib/nested-list/nested-list-popover/nested-list-popover.component.ts

```diff
@@ -32,6 +44,16 @@ export class NestedListPopoverComponent implements NestedListPopoverInterface {
     @ContentChild(NestedLinkDirective)
     linkDirective: NestedLinkDirective;
 
+    /** @hidden */
+    @ContentChild(NestedListContentDirective)
+    contentDirective: NestedListContentDirective;
+
+    /** @hidden
+     * For Internal Usage - Gets information about title, which should be displayed inside popover
+     */
+    @Input()
+    title: string = '';
```

Comment:
This is weird that `fd-nested-list-popover` needs a title. This is by Fiori3 design?

Reply:
You can take a look [sapui5](https://sapui5.hana.ondemand.com/#/entity/sap.tnt.SideNavigation/sample/sap.tnt.sample.SideNavigation). I wanted to follow this behaviour